docs/tutorials/01_portfolio_optimization.ipynb

```diff
@@ -280,13 +257,12 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "metadata": {},
    "source": [
     "## Solution using VQE\n",
```

Comment:
```suggestion
    "## Solution using SamplingVQE\n",
```

Reply:
Since we still have a VQE, I think its best to say SamplingVQE since that is the class which is used.